src/routes/MapExplorer/MapExplorer.js

```diff
@@ -192,14 +191,14 @@ export default class MapExplorer extends React.Component {
 
   render () {
     const {
+      currentUser,
       features,
       fetchPostsParam,
       filters,
       querystringParams,
       pending,
       routeParams,
```

Comment:
Passing `queryStringParams` and `routeParams` smells a little funny to me. If the component being rendered isn't part of the routing, and therefore doesn't have a `match.params` prop then for clarity's sake I might expect to pass only the specific keys (networkSlug, slug/communitySlug, etc) instead of the whole thing, starting from the connector in this component... That said, these leaf components are mostly route endpoints (and to me that is a good thing), and therefore should have access to `match.params`.  It is an option in those cases to retrieve these params directly within the component (and without connecting the component to Redux--which can be confusing as they are setup as selectors). It's just something I noticed. It can be confusing but worth keeping in mind, as some additional simplicity and clarity can be achieved.

Reply:
hmm yeah i hear you. the routeParams is used by PostCard and passed in all over the place to it so dont feel like doing that refactor right now, but i think i agree